### ISSUE TITLE
Replace 'filter' with 'action' in before_filter, prepend_before_filter

### ISSUE THREAD

**Issue by januszm** [schneems/wicked#203]
```sh
README.md:280:before_filter :set_locale
lib/wicked/controller/concerns/steps.rb:55:      prepend_before_filter(options) do
lib/wicked/wizard/translated.rb:8:        skip_before_filter :setup_wizard
lib/wicked/wizard/translated.rb:9:        before_filter      :setup_wizard_translated
lib/wicked/wizard.rb:13:        super "No step definitions have been supplied; if setting via `before_filter`, use `prepend_before_filter`"
lib/wicked/wizard.rb:29:      before_filter :setup_wizard
test/dummy/app/controllers/application_controller.rb:3:  before_filter :pull_out_locale
test/dummy/app/controllers/dynamic_different_steps_controller.rb:4:  before_filter :set_steps
test/dummy/app/controllers/dynamic_different_steps_controller.rb:5:  before_filter :setup_wizard
test/dummy/app/controllers/dynamic_steps_controller.rb:3:  prepend_before_filter :set_steps
```

**Comment by schneems** [comment]
This was done in #201 can you try master?

**Comment by januszm** [comment]
I just realized, it's on master but it is not in the released version of the gem. Can you release anytime soon?

**Comment by januszm** [comment]
THANK YOU! ;)

**Comment by schneems** [comment]
I thought I commented that I released 1.3.0, but I don't see it here. But it looks like you got the message based on your response. I'm re commenting to let other users know.

**Comment by januszm** [comment]
Ha, no I just checked github again and noticed v1.3.0 released couple of minutes ago (at that moment). Anyway thanks!
